src/main/java/com/cema/users/domain/User.java

```diff
@@ -0,0 +1,79 @@
+package com.cema.users.domain;
+
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.Date;
+
+public class User {
+
+    @ApiModelProperty(notes = "The username for this user, needed for login", example = "merlinds")
+    private String userName;
+    @ApiModelProperty(notes = "The name of the owner of this user", example = "Merlin")
```

Comment:
I think we should treat the user as the person itself, so "The name <s>of the owner</s> of this user"
Same applies to the other notes

Reply:
ok, valid.